lib/origen/log.rb

```diff
@@ -74,50 +88,56 @@ def level
       @level
     end
 
-    def debug(string = '')
+    def debug(string = '', msg_type = nil)
```

Comment:
Does this work ok if the string and msg_type and both missing?
Could you add a test for that to make sure?

i.e. this is allowed by the existing API and is used to print a blank line in the log:

~~~ruby
Origen.log.debug
Origen.log.debug "This is important so I added some padding"
Origen.log.debug
~~~